MediaBrowser.Providers/Tmdb/Movies/TmdbSearch.cs

```diff
@@ -86,36 +100,35 @@ private async Task<IEnumerable<RemoteSearchResult>> GetSearchResults(ItemLookupI
                 }
             }
 
+            // Ideally retrying alternatives should be done outside the search
+            // provider so that the retry logic can be common for all search
+            // providers
             if (results.Count == 0)
             {
-                // try with dot and _ turned to space
-                var originalName = name;
-
-                name = name.Replace(",", " ");
-                name = name.Replace(".", " ");
-                name = name.Replace("_", " ");
-                name = name.Replace("-", " ");
-                name = name.Replace("!", " ");
-                name = name.Replace("?", " ");
-
-                var parenthIndex = name.IndexOf('(');
-                if (parenthIndex != -1)
-                {
-                    name = name.Substring(0, parenthIndex);
-                }
+                name = parsedName.Name;
```

Comment:
why restart cleaning? I think you've already executed `_cleanNonWord.Replace()` on that...

Reply:
First time around: "[ABCD}" -> "ABCD" (some movie have these chars in their name)
Second time: "[ABCD]" -> "" (clean out some attributes)